go.mod

```diff
@@ -9,21 +9,24 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.8.1
-	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf
+	golang.org/x/term v0.0.0-20210615171337-6886f2dfbf5b
 	gotest.tools/v3 v3.0.3
 	k8s.io/api v0.20.7
 	k8s.io/apiextensions-apiserver v0.20.7
 	k8s.io/apimachinery v0.20.7
 	k8s.io/cli-runtime v0.20.7
 	k8s.io/client-go v0.20.7
 	k8s.io/code-generator v0.20.7
-	knative.dev/eventing v0.25.0
+	knative.dev/eventing v0.25.2
 	knative.dev/hack v0.0.0-20210622141627-e28525d8d260
+	knative.dev/kn-plugin-func v0.19.0
 	knative.dev/kn-plugin-source-kafka v0.25.0
 	knative.dev/networking v0.0.0-20210803181815-acdfd41c575c
-	knative.dev/pkg v0.0.0-20210803160015-21eb4c167cc5
-	knative.dev/serving v0.25.0
+	knative.dev/pkg v0.0.0-20210902173607-844a6bc45596
+	knative.dev/serving v0.25.1
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
+
+replace knative.dev/kn-plugin-func => github.com/openshift-knative/kn-plugin-func v0.19.1-0.20211103201756-d595897e6db0
```

Comment:
The version matches midstream: https://github.com/openshift-knative/kn-plugin-func/commit/d595897e6db0ef82a23035bd72bc0a382b2bf2d8